foundation/okfplugins/article_link/templates/article_link_plugin.html

```diff
@@ -3,7 +3,7 @@
         <img src="{% if instance.image %}{{ instance.image.url }}{% endif %}" alt="{{ instance.author }}">
     </div>
     <div class="description">
-        <strong class="block mb-5 text-h3">{{ instance.title }}</strong>
-        <span class="font-mono text-okfn-sm">{{ instance.author }}</span>
+        <strong class="block mb-5 text-h3 title">{{ instance.title }}</strong>
+        <span class="font-mono text-okfn-sm author">{{ instance.author }}</span>
```

Comment:
1) Are this two classes doing anything? I cannot see any effect while inspecting them.
2) Since we are using tailwind, can we use tailwind classes here instead?

Reply:
Thanks! 

These classes are defined in _article.css file and applies style to `:hover` and `:active` states for title. And a different  font style to author. So I re-used those classes. 
I took reference from here
https://ishigami.github.io/okfn_front/components.html